src/vellum/workflows/nodes/experimental/tool_calling_node/utils.py

```diff
@@ -59,8 +59,8 @@ def create_tool_router_node(
         function_name = function.__name__
         port_condition = LazyReference(
             lambda: (
-                ToolRouterNode.Outputs.results[0]["type"].equals("FUNCTION_CALL")
-                & ToolRouterNode.Outputs.results[0]["value"]["name"].equals(function_name)
+                node.Outputs.results[0]["type"].equals("FUNCTION_CALL")
+                & node.Outputs.results[0]["value"]["name"].equals(function_name)
```

Comment:
cc: @vincent0426 for tool calling node changes. Note that we want to associate direct node output references instead of the base class